tools/alpine/packages

```diff
@@ -0,0 +1,162 @@
+alpine-baselayout
```

Comment:
Potentially I could generate it from a shorter list by having another stage that installs all the packages in the list you give and then uses the list that is then installed, ie with dependencies, as the list to mirror...